draft-packages/tooltip/docs/Tooltip.stories.tsx

```diff
@@ -125,6 +125,9 @@ export const OverflowScroll = () => (
           position="below"
           display="inline-block"
           text="This should not get cropped"
+          // Normally, you'd specify a div by ID, but since this is only in storybook,
+          // using `body` is fine (I think).
+          portalSelector="body"
```

Comment:
I wasn't sure how to insert a `<div id="tooltips" />` into the dom, so I just used `body`